core/io/redirect.h

```diff
@@ -0,0 +1,64 @@
+/*************************************************************************/
+/*  redirect.h                                                                 */
+/*************************************************************************/
```

Comment:
Fix header alignment:

```suggestion
/*  redirect.h                                                           */
/*************************************************************************/
```

Reply:
Will fix with other comments comming in, so I don't flood the logs - thanks for checking and catching tho!